app/views/reciipes/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<p style="color: green"><%= notice %></p>
+
+<h1><%= @reciipe.name %></h1>
+<div class="recipelist-container detail">
+<div>
+<p><strong>Preparation Time:</strong> <%= @reciipe.preparation_time %> minutes</p>
+<p><strong>Cooking Time:</strong> <%= @reciipe.cooking_time %> minutes</p>
+<p><strong>Description:</strong> <%= @reciipe.description %></p>
+
+<div>
+  <%= link_to "Back to recipes", reciipes_path %>
+
+  <% if user_signed_in? && (current_user == @reciipe.user || @reciipe.public?) %>
+    <%= button_to "Destroy this recipe", @reciipe, method: :delete %>
+  <% end %>
+</div>
+</div>
+<div class="public">
+<%= form_with(model: @reciipe) do |form| %>
+  <% if user_signed_in? && current_user == @reciipe.user %>
+    <div>
+      <%= form.label :public, style: "display: block" %>
+      <%= form.check_box :public, onchange: "this.form.submit()" %>
+    </div>
+  <% end %>
+<% end %>
+</div>
+</div>
+
+<%= link_to "Add ingredient", new_reciipe_foods_reciipe_path(@reciipe), class: "btn" %>
+
+<div class="recipeActions">
+  <a href="/general_shopping_list">
+  <button type="button">Generate Shopping List</button>
```

Comment:
- [ ] Let's improve the design of the Recipe detail page by ensuring that our design closely matches the provided wireframe,  as we require close similarity in design for this project. 

**Your design:**        |  **Expected:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/MayPyone/Recipe-app/assets/12745474/744c89f1-f60f-488c-aec8-8abc66142a0b"> | <img width="800" alt="Home Page" src="https://github.com/MayPyone/Recipe-app/assets/12745474/431c6794-a116-4b03-baad-ff2def694b2c">